### DISCUSSION TITLE
Strange Azure PKCE??!!

### DISCUSSION THREAD

**Discussion by blendsdk** [panva/openid-client#681] [Q&A]
Hi all,

I am not sure whether this is a bug or a "hidden feature". But on a random basis I get an error from Azure stating PKCE code_verifier does not match! 

Have you guys ever encountered something like this?

`
azure AADSTS50148: The code_verifier does not match the code_challenge supplied in the authorization request for PKCE.
`

Cheers.

**Comment by panva** [comment]
That's something to bring up with your identity provider.

**Reply by blendsdk** [reply]
Thank you
